docs/developer-guide/rules.md

```diff
@@ -113,7 +113,89 @@ Stylelint has [utility functions](https://github.com/stylelint/stylelint/tree/ma
 Always use the:
 
 - `validateOptions()` utility to warn users about invalid options
-- `isStandardSyntax*` utilities before checking a node or string to ignore non-standard syntax
+- `isStandardSyntax*()` utilities before checking a node or string to ignore non-standard syntax
+- `report()` utility to report lint problems
+
+##### Location arguments for `report()`
+
+When using `report()`, you can specify the location of a problem in various ways:
+
+- only `node` - implicitly span the entire range of the given node
+- `word` - the first instance of a word in the serialized given node
+- `index` and `endIndex` offsets - an index range within the given node
+- `start` and `end` positions - a position range within the given node
```

Comment:
I wasn't sure how to describe these. Any suggestions?

Reply:
If you are talking about `start`/`end`, I would link to the PostCSS API directly.

i.e.
```suggestion
- `index` and `endIndex` offsets - an index range within the given node
- `start` and `end` positions - a [range](https://postcss.org/api/#range) within the given node
```